imageio/plugins/_freeimage.py

```diff
@@ -399,7 +400,30 @@ def error_handler(fif, message):
         
         # Make sure to keep a ref to function
         self._error_handler = error_handler
-        
+    
+    @property
```

Comment:
I like this pattern, we could generalize this into a generic property and and `load_library` method.